### ISSUE TITLE
I wrote a javascript client for RedisGraph

### ISSUE THREAD

**Issue by Jonahss** [RedisLabsModules/RedisGraph#335]
Hi!

I wasn't really satisfied with the javascript client in the README, didn't support promises and the returned result objects were a little hard to parse. So I wrote my own, built off of [ioredis](https://github.com/luin/ioredis) a pretty awesome js client for redis.

[jonahss/ioredisgraph](https://github.com/Jonahss/ioredisgraph)

Would you be ok with including my client as an option in the README? I can send up a PR.
Or any advice for additional features/functionality?

**Comment by swilly22** [comment]
Thank you @Jonahss, we really appreciate this contribution!

Two comment,
1. Would you mind fixing your repo desc: "Node.js client for RedisGrap, by wrapping ioredis module"
to "Node.js client for RedisGrap**h**, by wrapping ioredis module"

Can you please set a license to ioredisgraph?

**Comment by Jonahss** [comment]
Definitely! Thanks for taking a look.
Changed description and added ISC license.
